src/js/pages/settings/UISettingsPage.js

```diff
@@ -83,8 +100,58 @@ class UISettingsPage extends React.Component {
                 }
               />
             </ConfigurationMapSection>
+            <ConfigurationMapSection>
+              <ConfigurationRow
```

Comment:
Non-blocking suggestion: can we move this into the same `ConfigurationMapSection` as the Language setting? I think it looks a little nicer that way.

![Screen Shot 2019-11-06 at 2 17 28 PM](https://user-images.githubusercontent.com/2313998/68330047-88510580-00a0-11ea-9b11-05cedee451e6.png)